src/settings_gui.cpp

```diff
@@ -163,6 +163,18 @@ static const std::map<int, StringID> _scale_labels = {
 	{  500, STR_GAME_OPTIONS_GUI_SCALE_5X },
 };
 
+static const std::map<int, StringID> _volume_labels = {
```

Comment:
Why is this necessary now, when it was not previously? Does it need some doxyment?

Reply:
To be honest it's not absolutely needed, but having the labels makes the two sliders look better (IMHO) when placed directly together.